sklearn/svm/tests/test_svm.py

```diff
@@ -707,7 +708,7 @@ def test_bad_input():
     # predict with sparse input when trained with dense
     clf = svm.SVC().fit(X, Y)
     with pytest.raises(ValueError):
-        clf.predict(sparse.lil_matrix(X))
+        clf.predict(csr_container(X))
```

Comment:
Please use a `lil_container` from `LIL_CONTAINERS` to preserve the original intention of the test.